cpu/stm32/Makefile.include

```diff
@@ -43,6 +43,32 @@ ifneq (,$(CCMRAM_LEN))
   LINKFLAGS += $(LINKFLAGPREFIX)--defsym=_ccmram_length=$(CCMRAM_LEN)
 endif
 
-VECTORS_O ?= $(BINDIR)/stm32_vectors/vectors_$(CPU_FAM).o
+ifneq (,$(filter STM32F030x4,$(CPU_LINE)))
+  VECTORS_O ?= $(BINDIR)/stm32_vectors/$(CPU_LINE).o
+else
+  VECTORS_O ?= $(BINDIR)/stm32_vectors/gen_$(CPU_LINE).o
+  VECTORS_FILE = $(RIOTCPU)/stm32/vectors/gen_$(CPU_LINE).c
+  BUILDDEPS += $(VECTORS_FILE)
+
+  # Add this define to skip the inclusion of the system_stm32xxxx.h header files
+  # which are only used for STM32 system includes and not of interest for RIOT.
+  CFLAGS += -D__SYSTEM_STM32$(call uppercase,$(CPU_FAM))XX_H
+
+  STM32CMSIS_INCLUDE_DIR = $(RIOTCPU)/stm32/include/vendor/cmsis/$(CPU_FAM)/Include
+  STM32FAM_INCLUDE_FILE = $(STM32CMSIS_INCLUDE_DIR)/stm32$(CPU_FAM)xx.h
+  INCLUDES += -I$(STM32CMSIS_INCLUDE_DIR)
+
+  STM32IRQS_INCLUDE_FILE = $(RIOTCPU)/stm32/include/irqs/$(CPU_FAM)/irqs.h
+  BUILDDEPS += $(STM32IRQS_INCLUDE_FILE)
+
+$(STM32FAM_INCLUDE_FILE):
```

Comment:
If you add this it will always call make on the pkg, but if there has been no change then nothing will happen otherwise it will update it.

```suggestion
$(STM32FAM_INCLUDE_FILE): FORCE
```

Reply:
Fixed